packages/core/src/challenger/submitAssertionToReferee.ts

```diff
@@ -16,15 +18,39 @@ export async function submitAssertionToReferee(
     challengerBlsSecretKey: string,
     assertionId: number,
     assertionNode: AssertionNode,
-    signer: ethers.Signer
+    signer: ethers.Signer,
+    currentChallenge: Challenge
```

Comment:
If we only use the assertionId from the challenge we should not pass the whole challenge object into this function, it makes it harder to follow.

This should be the `predecessorAssertionId` to get back in sync to what it's on the Referee.

Reply:
Done!